src/kml/src/main/resources/org/geoserver/kml/icon_style_patch.xsl

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright (C) 2017 - Open Source Geospatial Foundation. All rights reserved.
+ This code is licensed under the GPL 2.0 license, available at the root
+ application directory.
+ -->
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xalan="http://xml.apache.org/xalan" xmlns:kml="http://www.opengis.net/kml/2.2">
+    <xsl:output indent="yes" method="xml" standalone="yes" xalan:indent-amount="4" />
```

Comment:
Explicit xalan reference... wondering if it might cause issues in specific web containers forcing their own XML technology. Do you have any clue?
No change required, just wondering.